Sodium/Sign.swift

```diff
@@ -38,7 +38,7 @@ public class Sign {
         return KeyPair(publicKey: PublicKey(data: pk!), secretKey: SecretKey(data: sk!))
     }
     
-    public func keyPair(seed seed: NSData) -> KeyPair? {
+    public func keyPair(#seed: NSData) -> KeyPair? {
```

Comment:
xcode 7 has not yet released. 
what is correct function signature will be in this case?
```public func keyPairWithSeed(seed: NSData) -> KeyPair?``` ?